src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Color;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int RANDOM_NUMBER_FIGURE_RANGE = 4;
+    private static final double RANDOM_BOUND = 100.00;
+    private static final double DEFAULT_CIRCLE_VALUE = 10.0;
+    private final Random random = new Random();
+
```

Comment:
```suggestion
```

Reply:
yeah, sometimes i wrote like this, cuz i want to make the code more readable. Split different sections, like initialization and etc.